py-polars/pyproject.toml

```diff
@@ -4,10 +4,40 @@ build-backend = "maturin"
 
 [project]
 name = "polars"
+description = "Blazingly fast DataFrame library"
+readme = "README.md"
+authors = [
+    {name = "Ritchie Vink", email = "ritchie46@gmail.com"},
+]
+license = {text = "MIT"}
```

Comment:
should probably be `license = { file = "LICENSE" }`

Reply:
I believe either is fine, see:
https://peps.python.org/pep-0621/#license

But it's just a bit more accurate to specify the actual license file.